src/main/resources/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "rdp",
+  "private": true,
+  "scripts": {
+    "build": "webpack --config webpack.config.js"
+  },
+  "dependencies": {
+    "bootstrap": "4.0.0",
+    "bootstrap-select": "1.13.14",
+    "datatables.net": "1.12.1",
+    "datatables.net-bs4": "1.12.1",
+    "jquery": "3.2.1",
+    "jquery-ui": "1.12.1",
+    "open-iconic": "1.1.1",
+    "popper.js": "1.12.9"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.18.5",
+    "@babel/preset-env": "^7.18.2",
+    "babel-loader": "^8.2.5",
+    "source-map-loader": "^4.0.0",
+    "webpack": "^5.73.0",
+    "webpack-cli": "^4.10.0"
+  },
+  "browserslist": [
+    "> 1%"
```

Comment:
This configuration should be carefully reviewed with what we expect from our userbase.

Reply:
We should use `defaults` here.